_pytest/python.py

```diff
@@ -131,6 +131,7 @@ def pytest_namespace():
     raises.Exception = pytest.fail.Exception
     return {
         'raises': raises,
+        'not_raises': NotRaisesContext,
```

Comment:
oh, an additonal note, why do we make it a callable?

i believe making it a singleton context-manager named after exactly the context its going to encompass is the best we can do,

else we might see people wanting to creep in more specific behavior causing grief down the line